### ISSUE TITLE
Update to support Pydantic v2

### ISSUE THREAD

**Issue by mlisthenewcool** [fastapi-users/fastapi-users#1246]
FastAPI v0.100+ uses Pydantic v2.

It seems to work out-of-the-box with fastapi-users v12 but I get a deprecation warning :  

```
python3.11/site-packages/pydantic/_internal/_config.py:261: UserWarning: Valid config keys have changed in V2:
* 'orm_mode' has been renamed to 'from_attributes'
````

**Comment by mlisthenewcool** [comment]
Unfortunately, I was too fast to say it will work. That warning raises an Exception :
```pydantic.errors.PydanticUserError: You must set the config attribute `from_attributes=True` to use from_orm```

**Comment by frankie567** [comment]
Released in [v12.1.0](https://github.com/fastapi-users/fastapi-users/releases/tag/v12.1.0) 🎉
